pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -3066,3 +3083,371 @@ fn multiple_custodian_settlement() {
             );
         });
 }
+
+// ----------------------------------------- Confidential transfer tests -----------------------------------
+
+pub fn gen_account(
+    tx_id: u32,
+    seed: [u8; 32],
+    token_name: &[u8],
+    valid_asset_ids: Vec<AssetId>,
+) -> (SecAccount, PubAccountTx) {
+    // These are the encryptions keys used by MERCAT and are different from the signing keys
+    // that Polymesh uses.
+    let mut rng = StdRng::from_seed(seed);
+    let elg_secret = ElgamalSecretKey::new(Scalar::random(&mut rng));
+    let elg_pub = elg_secret.get_public_key();
+    let enc_keys = EncryptionKeys {
+        pblc: elg_pub.into(),
+        scrt: elg_secret.into(),
+    };
+
+    let asset_id = AssetId {
+        id: *Ticker::try_from(token_name).unwrap().as_bytes(),
+    };
+    let asset_id_witness = CommitmentWitness::from((asset_id.clone().into(), &mut rng));
+    let secret_account = SecAccount {
+        enc_keys,
+        asset_id_witness,
+    };
+    let valid_asset_ids = convert_asset_ids(valid_asset_ids);
+    let mercat_account_tx = AccountCreator {}
```

Comment:
I think it `AccountCreator` requires `{}` (which looks a bit unusual) because it is defined with those. Is it possible to define it without parentheses?
`pub struct AccountCreator;`

Reply:
Nice, did not know that. This requires a change in the cryptography library. I'll make the change in another PR.

Reply:
https://polymath.atlassian.net/browse/CRYP-173